cop.h

```diff
@@ -109,27 +109,36 @@ typedef struct jmpenv JMPENV;
  */
 
 #define dJMPENV		JMPENV cur_env
+#ifdef DEBUGGING
+#define iDECL int i = 0
+#else
+#define iDECL
+#endif
 
-#define JMPENV_PUSH(v) \
+#define JMPENV_PUSH(v)                                                  \
     STMT_START {							\
-        DEBUG_l({							\
-            int i = 0; JMPENV *p = PL_top_env;				\
+        iDECL;                                                          \
```

Comment:
This could just be `PERL_DEB(int i = 0);`

Reply:
Oh, nice, I didnt know that one. Willdo.